src/opex_dashboard/templates/api-management_queries/availability.kusto

```diff
@@ -1,7 +1,7 @@
 {% load stringify uri_to_regex %}
 let threshold = {{ threshold|default:"0.99" }};
 AzureDiagnostics
-| where url_s matches regex "{{ endpoint|uri_to_regex }}"
+| where url_s matches regex "{{ base_path|default:""|add:endpoint|uri_to_regex }}"
```

Comment:
[**add**](https://docs.djangoproject.com/en/4.2/ref/templates/builtins/#add) is meant for integeres, what about:
```suggestion
| where url_s matches regex "{{ base_path|default:""}}{{ endpoint|uri_to_regex }}"
```

Reply:
As for now add operation is concatenating strings. I used this approach so that the base path + endpoint string concatenation is passed to the uri-to-regex function, with this modification only the endpoint part will be processed by the uri-to-regex function, right?

Reply:
Fixed with https://github.com/pagopa/opex-dashboard/pull/46/commits/135679761af2bcb3a0cac9c3d626210dd3401a5d

Changed :add to [:add_str](https://github.com/pagopa/opex-dashboard/blob/04f5fd5a61d1234490e438780f2e525f23cd309a/src/opex_dashboard/tags/add_str.py#L6-L9) in order to avoid string to in coercion